Job sync status (#6322)
braze-inc/braze-docs: _docs/_api/endpoints/cdi/get_job_sync_status.md

}
```

> | job_status | explanation |
> | --- | --- |
> | Running | The job is current running |
> | Success | All rows synced successfully |
> | Partial | Some rows failed to sync due to errors |
> | Error | No rows were synced |
> 

Reviewer: ```suggestion
| job_status | Explanation |
| --- | --- |
| Running | The job is currently running. |
| Success | All rows synced successfully. |
| Partial | Some rows failed to sync due to errors. |
| Error | No rows were synced. |
{: .reset-td-br-1 .reset-td-br-2}
```

> | --- | --- |
> | Running | The job is currently running. |
> | Success | All rows synced successfully. |
> | Partial | Some rows failed to sync due to errors. |
> | Error | No rows were synced. |
> {: .reset-td-br-1 .reset-td-br-2}

Reviewer: ```suggestion
| --- | --- |
| `Running` | The job is currently running. |
| `Success` | All rows synced successfully. |
| `Partial` | Some rows failed to sync due to errors. |
| `Error` | No rows were synced. |
{: .reset-td-br-1 .reset-td-br-2}
```

> {: .reset-td-br-1 .reset-td-br-2}

Reviewer: ```suggestion
{: .reset-td-br-1 .reset-td-br-2}

```
